app/routes/models.js

```diff
@@ -1,77 +1,88 @@
 const Joi = require('joi')
-const { addModel, getModels, getModel } = require('../storage/repos/models')
+const { addModel, getModels, getAllModels, getModel } = require('../storage/repos/models')
 
-module.exports = [
-  {
-    method: 'GET',
-    path: '/models',
-    options: {
-      validate: {
-        query: Joi.object({
-          deploymentVendor: Joi.string().required()
-        })
-      }
-    },
-    handler: async (request, h) => {
-      const models = await getModels(request.query.deploymentVendor)
+module.exports = [{
+  method: 'GET',
+  path: '/models',
+  options: {
+    validate: {
+      query: Joi.object({
+        deploymentVendor: Joi.string().required()
+      })
+    }
+  },
+  handler: async (request, h) => {
+    const models = await getModels(request.query.deploymentVendor)
 
-      if (models.length === 0) {
-        return h.response().code(204)
-      }
+    if (models.length === 0) {
+      return h.response().code(204)
+    }
 
-      return h.response(models).code(200)
+    return h.response(models).code(200)
+  }
+},
+{
+  method: 'GET',
```

Comment:
is a separate route necessary here? the getModels() and getAllModels() functions look very similar. Would it not be more readable to have a single route with an optional deploymentVendor query param. When present only return those models and if not return all?

Reply:
The JSON that is returned by both methods is different, `getModels()` is a simple array of models, whereas `getAllModels()` returns an array of vendors each with a nested array of models, so best to keep them separate

Reply:
`getModels()` was the original behaviour, agree that `getModelsByVendor` might be more appropriate

Reply:
Commit [2952eca](https://github.com/DEFRA/coreai-mcu-prompts/pull/3/commits/2952ecaac03b365f346007a43ced1fe2d35737c6) renames routes